website/astro.config.mjs

```diff
@@ -0,0 +1,36 @@
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+
+// https://astro.build/config
+export default defineConfig({
+	integrations: [
```

Comment:
Please reformat all files to comply with the `.editorconfig`: Two spaces for indentation, with a newline at the end of the file.

https://github.com/facebook/jscodeshift/blob/74c6af3338f6d3520cf3fff37049f34b4d15efc3/.editorconfig#L5-L7